MODULE.bazel

```diff
@@ -8,7 +8,7 @@ module(
 
 # Lower-bounds (minimum) versions for direct runtime dependencies.
 # Do not bump these unless rules_js requires a newer version to function.
-bazel_dep(name = "aspect_bazel_lib", version = "2.14.0")
+bazel_dep(name = "aspect_bazel_lib", version = "2.16.0")
```

Comment:
Do we need this upgrade?

Reply:
Yeah see above, I checked it was the earliest one. We could use tar.bzl directly instead I suppose?